zkevm-circuits/src/bytecode_circuit/bytecode_unroller.rs

```diff
@@ -157,9 +156,9 @@ impl<F: Field> Config<F> {
                 ),
             );
             cb.require_equal(
-                "hash_input_rlc := hash_input_rlc_prev * randomness + byte",
+                "hash_input_rlc := hash_input_rlc_prev * challenges.keccak_input + byte",
                 meta.query_advice(hash_input_rlc, Rotation::cur()),
-                meta.query_advice(hash_input_rlc, Rotation::prev()) * randomness.clone()
+                meta.query_advice(hash_input_rlc, Rotation::prev()) * challenges.keccak_input()
```

Comment:
It seems that `hash_input_rlc` depends on the challange.  Should it be an advice in `SecondPhase`?

Reply:
Yes, it should be in `SecondPhase`, will test with real prover then submit a fix soon.